internal/cache/cache.go

```diff
@@ -0,0 +1,111 @@
+package cache
+
+import (
+	"sync"
+	"time"
+)
+
+//go:generate mockgen -source=cache.go -package=cache -destination=mock_cache.go
+
+type Cache interface {
+	DeleteExpired()
+	Get(key string) (interface{}, bool)
+	Set(key string, value interface{})
+	StopCleanup()
+}
+
+type Item struct {
```

Comment:
Does this type need to be exported?

Reply:
No, we don't need it exported. It's just a leftover while testing this code, I'll change it to an non-exported field. Thanks for spotting this :+1: